src/components/adminpanel/adminpanel.jsx

```diff
@@ -23,71 +22,45 @@ class AdminPanel extends React.Component {
     }
     render () {
         if (!this.props.isAdmin) return false;
-
-        if (this.state.showPanel) {
-            return (
-                <div
-                    className="visible"
-                    id="admin-panel"
-                >
+        return (
+            <div
+                className={classNames(
+                    'admin-panel', this.props.className, {
+                        hidden: !this.state.showPanel
+                    }
+                )}
+            >
+                {this.state.showPanel ? (
```

Comment:
I couldn't see the panel content when open, because scratchr2 gave a 404. I assume this is waiting for a change on scratchr2's side? 

Url: http://localhost:8080/scratch2/2000000157/adminpanel/

![image](https://user-images.githubusercontent.com/3431616/49583270-a7419780-f925-11e8-857f-11bf6b263f4e.png)